content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -4,274 +4,291 @@ Start receiving tickets
+on the team's settings page. To change a *Helpdesk* team's email alias, navigate to
+:menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams` and click on a team name to open
+its settings page. Scroll to :menuselection:`Channels --> Email Alias`. In the :guilabel:`Alias`
+field, type the desired name for the team's email alias.
 
 .. image:: receiving_tickets/receiving-tickets-email-alias.png
    :align: center
-   :alt: View of the settings page of a helpdesk team emphasizing the email alias feature in Odoo
+   :alt: View of the settings page of a Helpdesk team emphasizing the email alias feature in Odoo
          Helpdesk.
 
-When an email is received, the subject line from the email becomes the title of a new *Helpdesk*
-ticket. The body of the email is also added to the ticket under the :guilabel:`Description` tab and
-in the ticket's :guilabel:`Chatter` section.
-
-.. important::
-   The configuration steps outlined above are for **Odoo Online** and **Odoo.sh** databases. For
-   **On-premise** databases, additional configuration for custom email servers and email aliases
-   may be required.
-
-Website Form
-------------
-
-Enabling the *Website Form* setting adds a new page to the website with a customizable
-form. A new ticket is created once the required form fields are filled out and submitted.
-
-To activate the website form, navigate to a team's settings page under :menuselection:`Configuration
---> Teams`. Find the :guilabel:`Website Form` feature under the :guilabel:`Channels` section, and
-check the box.
-
-After the feature is activated, click the :guilabel:`Go to Website` smart button at the top of the
-that :guilabel:`Teams` settings page to view and edit the new website form, which is created
-automatically by Odoo.
-
 .. note::
-   After enabling the website form, the *Teams* settings page may need to be refreshed before the
-   *Go to Website* smart button appears.
-
-   As well, if a *Help Center* is published, the smart button will navigate there first. Simply click
-   the :guilabel:`Contact Us` button at the bottom of the forum to navigate to the ticket submission
-   form.
+   Custom email domains are **not** required in order to use an email alias, however, they can be
+   configured through the *Settings* app. If the database does not have a custom domain already
+   configured, click :guilabel:`Configure a custom domain` to be redirected to the
+   :guilabel:`Settings` page. From there, enable :guilabel:`Custom Email Servers`.
 
-.. image:: receiving_tickets/receiving-tickets-go-to-website.png
-   :align: center
-   :alt: View of the settings page of a helpdesk team emphasizing the Go to Website button in
-         Odoo Helpdesk.
+When an email is received, the subject line becomes the title of a new *Helpdesk* ticket. The body
+of the email is also added to the ticket under the :guilabel:`Description` tab and in the ticket's
+:guilabel:`Chatter`.
```

Comment:
You are right, and I think this will be part of the "Odoo specific" guidelines we discussed. In this case, since it is referring to the chatter on a ticket and not instructing the user to interact with the element directly, I will italicize it instead of using the guilabel